vm/actor/src/builtin/miner/policy.rs

```diff
@@ -63,6 +63,7 @@ pub fn check_supported_proof_types(proof: RegisteredSealProof) -> bool {
 /// Maximum duration to allow for the sealing process for seal algorithms.
 /// Dependent on algorithm and sector size
 pub fn max_seal_duration(proof: RegisteredSealProof) -> Option<ChainEpoch> {
+    // TODO needs to be updated/removed for v2
```

Comment:
Not sure what this comment means? Is this also waiting for miner v2 upgrade you mean?

Reply:
yeah, I didn't want to expand the scope of this PR, but the power changes broke this reference

Reply:
Cool